### ISSUE TITLE
Magic item effects do not work

### ISSUE THREAD

**Issue by Ulitmatex** [Muttley/foundryvtt-shadowdark#316]
![image](https://user-images.githubusercontent.com/77658242/231333629-64e3e643-1b6d-4bff-a73c-f19d943b3053.png)
When creating a magic item and trying to add one of the pre-selected effects you are unable to save a chosen effect to apply.

**Comment by bakbakbakbakbak** [comment]
Are you trying to create this item on an actor? That won't work due to limitations from Foundry.

If you try to create an item in the Items sidebar, this is an issue! I just tried this on my end, and I can create it - so if this is the case, I need to know more.

**Comment by Ulitmatex** [comment]
Im trying to create it on an actor i guess. Was using a pre-made weapon dragging it onto an actor and trying to make it a magic weapon and give it effects

**Comment by bakbakbakbakbak** [comment]
Yeah, unfortunately that is not supported by Foundry v10. It should be possible to do this in the future, but for now it is not. If you just drag the item to the Item Sidebar, modify it, and then drag it back - it will work.

Closing this, and will implement #317 for v10 so this isn't a possible scenario.
